pkg/modules/testdata/cr_powerstore_resiliency.yaml

```diff
@@ -18,7 +18,15 @@ spec:
       configVersion: "v1.6.0"
       enabled: true
       components:
-      - name: podmon
+      - name: podmon-controller
+        image: dellemc/podmon:v1.6.0
+        envs:
+          - name: "X_CSI_PODMON_API_PORT"
+            value: "8083"
+          - name: "X_CSI_PODMON_ARRAY_CONNECTIVITY_POLL_RATE"
+            value: "60"
+        args: ["arrayConnectivityPollRate=60"]
+      - name: podmon-node
         image: dellemc/podmon:v1.6.0
```

Comment:
Hi @sakshi-garg1 Can you extend this manifest file by passing args to every components, what we have added in sample files.
I was just going through with the code and to me it seems to be that after adding those args, our test coverage will be more.

Example:
      components:
        - name: podmon-controller
          args:
            - "--csisock=unix:/var/run/csi/csi.sock"
            - "--labelvalue=csi-powerstore"
            - "--arrayConnectivityPollRate=60"
            - "--driverPath=csi-powerstore.dellemc.com"
            - "--mode=controller"
            - "--skipArrayConnectionValidation=false"
            - "--driver-config-params=/powerstore-config-params/driver-config-params.yaml"
            - "--driverPodLabelValue=dell-storage"
            - "--ignoreVolumelessPods=false"
        - name: podmon-node
          envs:
            # podmonAPIPort: Defines the port to be used within the kubernetes cluster
            # Allowed values: Any valid and free port (string)
            # Default value: 8083
            - name: "X_CSI_PODMON_API_PORT"
              value: "8083"
          args:
            - "--csisock=unix:/var/lib/kubelet/plugins/csi-powerstore.dellemc.com/csi_sock"
            - "--labelvalue=csi-powerstore"
            - "--arrayConnectivityPollRate=60"
            - "--driverPath=csi-powerstore.dellemc.com"
            - "--mode=node"
            - "--leaderelection=false"
            - "--driver-config-params=/powerstore-config-params/driver-config-params.yaml"
            - "--driverPodLabelValue=dell-storage"
            - "--ignoreVolumelessPods=false"

Reply:
I have taken care of this in my latest PR so closing this one.